x-pack/plugins/siem/public/components/page/hosts/events_table/index.tsx

```diff
@@ -136,8 +137,8 @@ const getEventsColumns = (startDate: number): Array<Columns<EcsEdges>> => [
               queryMatch: {
                 displayField: 'host.name',
                 displayValue: hostName,
-                field: 'host.id',
-                value: node.host!.id!,
+                field: hostId ? 'host.id' : 'host.name',
```

Comment:
if we can have duplicate of  host id, I feel like we should just use `host.name`

Reply:
Yea, I went ahead and updated it to be just `host.name` to avoid confusions now.